### DISCUSSION TITLE
请问为什么当批量使用openpose的时候，如果源图片过多就会爆显存

### DISCUSSION THREAD

**Discussion by yaolunmao** [continue-revolution/sd-webui-animatediff#490] [Q&A]
> 请原谅我用中文发布讨论，因为我的英文不太好，而且我看到作者是懂中文的。如果给您造成了困扰，在这里先说声抱歉

我发现在所有参数都一致的情况下，同时开启webui的animatediff和controlnet里面批量处理openpose的功能时，如果我的源图片只有10张，就可以按照期望成功生成。但是当我的源图片有20张的时候显存就会溢出。我期望的效果是源图片越多处理的时间相对来说就越长，而不是占用更多的资源

**Comment by wood-123456** [comment]
老哥你把code放上來我看看?

**Comment by wood-123456** [comment]
沒有xformer顯存就需要12G
